<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAQ_Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gBGraph">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Graph Type</string>
       </property>
       <widget class="QCheckBox" name="cBOscill">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Oscillogram</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cBStereo">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>40</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Stereogram</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>901</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="gBDataSrc">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Data Source</string>
       </property>
       <widget class="QTextEdit" name="textFilePath">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>581</width>
          <height>41</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSelect">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>40</y>
          <width>93</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelFilePath">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>File Path</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>901</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="gBParam">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <widget class="QCheckBox" name="cBSChannels">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>110</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Channels</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cBAChannels">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>All Channels</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelArrange">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Selected sensor data from 18 channels</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelRenderPoints">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Displaying Points Per Frame</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAllChannels">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>110</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>All Channel List</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPointsRange">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>50</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>in  [20, 1000]</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>50</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>901</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="btnRender">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Render</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCancel">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>20</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Data_File">
   <property name="text">
    <string>Select Data File</string>
   </property>
  </action>
  <action name="actionReal_Time_Data">
   <property name="text">
    <string>Real-Time Data</string>
   </property>
  </action>
  <action name="actionParameter">
   <property name="text">
    <string>Parameter Configuration</string>
   </property>
  </action>
  <action name="actionOscillogram">
   <property name="text">
    <string>Oscillogram</string>
   </property>
  </action>
  <action name="actionStereogram">
   <property name="text">
    <string>Stereogram</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart_Rendering">
   <property name="text">
    <string>Start Rendering</string>
   </property>
  </action>
  <action name="actionStop_Rendering">
   <property name="text">
    <string>Stop Rendering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
